### DISCUSSION TITLE
How to train multiple concepts in one dreambooth lora sdxl model via advanced diffusion training script in diffusers' example

### DISCUSSION THREAD

**Discussion by joey0922** [huggingface/diffusers#7815] [Q&A]
I used diffusers' example [train_dreambooth_lora_sdxl_advanced.py](https://github.com/huggingface/diffusers/blob/main/examples/advanced_diffusion_training/train_dreambooth_lora_sdxl_advanced.py) to train single concept lora model, and it turns out well. I haved scanned the script and found that it seems able to train multiple concepts, but I didn't find any instruction how to do it. I just want to train some different cars in to one dreambooth lora sdxl model. Could anybody help me out of this? Really appreciate it.

**Comment by joey0922** [comment]
I figure it out myself

**Reply by techUdayMungalpara** [reply]
can you share how to do it
